recipes/react-ssr/webpack.config.js

```diff
@@ -29,13 +29,19 @@ function createConfig(opts) {
     name,
     mode: optimize ? 'production' : 'development',
     entry: isServer
-      ? { server: './server/main.js' }
+      ? { server: [...(optimize ? [] : ['webpack-plugin-serve/node-client']), './server/main.js'] }
```

Comment:
For some reason when node-client entry is placed after main server entry nothing works

Reply:
that's very odd

Reply:
Yeah, when node-client entry is the last one, then server bundle (server.js) doesn't export renderer function.

Reply:
wouldn't it be `{ server: [...(optimize ? ['./server/main.js'] : ['webpack-plugin-serve/node-client']), './server/main.js'] }`?

main has to be on the entry anyways, not only on dev

Reply:
Yes, main is always included. Only 'webpack-plugin-serve/node-client' is included conditionally.

Reply:
oh,  i missed a bracket! Those brackets always confuses me.

Reply:
well, actually this is quite logical behaviour. Webpack combines two modules and uses module.exports statement of the last one.